modules/ROOT/pages/set-payload-transformer-reference.adoc

```diff
@@ -55,4 +55,4 @@ The next example sets the message payload to "Hello, World" appending today's da
 == See Also
 
 * xref:about-mule-message.adoc[Mule Message Structure]
-* xref:dataweave.adoc[About DataWeave]
\ No newline at end of file
+* xref:dataweave.adoc[DataWeave Language]
```

Comment:
In most cases, you changed DataWeave language to DataWeave script, but I don't know enough to know if they are two different things.

Reply:
It's kind of confusing due to the file names which were retained to avoid breaking people's bookmarks (a questionable practice), but xref:dataweave.adoc[DataWeave Language] should contain "DataWeave Language" and something very similar xref:dataweave-language-introduction.adoc[DataWeave Scripts] contains "DataWeave Scripts".